src/wallet/coincontrol.h

```diff
@@ -26,6 +26,9 @@ enum class CoinType
     MAX_COIN_TYPE = ONLY_COINJOIN_COLLATERAL,
 };
 
+//! Default for -avoidpartialspends
+static constexpr bool DEFAULT_AVOIDPARTIALSPENDS = false;
```

Comment:
17518 what's stopping us from being able to remove this circular depends

Reply:
eliminated